lib/const.rb

```diff
@@ -37,12 +37,14 @@
 # does not compatible with the kernel architecture natively
 QEMU_EMULATED = !CPU_SUPPORTED_ARCH.include?(KERN_ARCH)
 
-# This helps with virtualized builds on aarch64 machines
-# which report armv8l when linux32 is run.
-ARCH = KERN_ARCH.eql?('armv8l') ? 'armv7l' : KERN_ARCH
-
-# This doesn't actually represent the userspace architecture (if the userspace architecture is aarch64), but we behave as if it does
-USER_SPACE_ARCH = ARCH.eql?('aarch64') ? 'armv7l' : ARCH
+# This helps with virtualized builds on aarch64 machines which report armv8l when linux32 is run.
+# We also report aarch64 machines as armv7l for now, as we treat them as if they were armv7l.
+# When we have proper aarch64 support, remove this.
+ARCH = if KERN_ARCH == 'armv8l' || KERN_ARCH == 'aarch64'
```

Comment:
I like this here...

Reply:
`ARCH = %w[aarch64 armv8l].include?(KERN_ARCH)`?

Reply:
There's no amazing solution for this, but it is nicer to have it in one line.